cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/SpannerStructuredTableAdmin.java

```diff
@@ -230,7 +233,15 @@ private StructuredTableSchema loadSchema(StructuredTableId tableId) throws Table
       throw new TableNotFoundException(tableId);
     }
 
-    return new StructuredTableSchema(tableId, fields, primaryKeys, indexes);
+    List<String> primaryKeys = primaryKeysOrderMap.keySet()
+      .stream()
+      .sorted()
```

Comment:
What the sorting is for? If you need the keys sorted, can we use a SortedMap instead?

Reply:
updated with TreeMap